.gitignore

```diff
@@ -3,6 +3,10 @@ installs
 *.log
 node_modules/
 
+# Added for pedestal framework test
```

Comment:
Oh...I see you did that already. I guess these couldn't go in there for some reason!